build.gradle

```diff
@@ -129,13 +130,13 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web'
     compile("org.springframework.boot:spring-boot-starter-actuator")
-    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
-    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
-    compile group: 'org.liquibase', name: 'liquibase-core', version: '3.8.7'
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: '3.0.0'
+    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '3.0.0'
```

Comment:
Same as other PR, please check that Swagger still works on 3.x as believe there will be other changes required.

Reply:
correct, already mentioned to Henry that we need to check we didn't break Swagger

Reply:
swagger works now..